src/Server/Server.fs

```diff
@@ -112,19 +98,19 @@ let topLevelRouter = router {
     forward "/api" (fun next ctx ->
         let cString = 
             let settings = ctx.GetService<IConfiguration>()
-            settings.["Swate:ConnectionString"]
+            settings.["Swate:LocalConnectionString"]
```

Comment:
One could argue that we dont need the local db connection string as a secret at all, as the dev can set that pw anyways, and the db is local. Lets leave it there for now, so we dont need to change that much at once